pkg/chartutil/requirements.go

```diff
@@ -126,7 +125,7 @@ func getAliasDependency(charts []*chart.Chart, aliasChart *chart.Dependency) *ch
 }
 
 // ProcessRequirementsEnabled removes disabled charts from dependencies
-func ProcessRequirementsEnabled(c *chart.Chart, v map[string]interface{}) error {
+func ProcessRequirementsEnabled(c *chart.Chart, v []byte) error {
```

Comment:
quick fly-by comment here and haven't taken too close of a look at the rest of the code yet, but this would be considered an API breaking change. Any chance we can revert this back? Users outside of Helm rely on `pkg/chartutil` so this would definitely break some user-facing client APIs, and therefore our semver contract.

Reply:
Now I found the real issue - requirements.go used the wrong YAML package which could not marshal the data.